HACKING.md

```diff
@@ -168,6 +168,14 @@ For coding style, we follow [PEP 8](https://peps.python.org/pep-0008/) as well a
 as the [ops library reference](https://ops.readthedocs.io/en/latest/reference/index.html)
 is automatically generated from Python docstrings.
 
+Changes should include tests. Where reasonable, prefer to write 'Scenario' tests via [ops.testing](https://ops.readthedocs.io/en/latest/reference/ops-testing.html) over legacy [ops.testing.Harness](https://ops.readthedocs.io/en/latest/reference/ops-testing-harness.html) tests. Note that these tests will require the `ops-scenario` package to be installed. Running tests with `tox -e unit` will take care of this automatically. These tests should be decorated so that they're skipped if `ops-scenario` isn't installed, like so:
```

Comment:
Couple of tweaks. And I don't think we need to mention that ops-scenario is required, as it's required now, and it's taken care of automatically as you say.

```suggestion
Changes should include tests. Where reasonable, prefer to write 'Scenario' tests using [ops.testing](https://ops.readthedocs.io/en/latest/reference/ops-testing.html) instead of legacy [ops.testing.Harness](https://ops.readthedocs.io/en/latest/reference/ops-testing-harness.html) tests. These tests should be decorated so that they're skipped if `ops-scenario` isn't installed, like so:
```

Reply:
Hmm, isn't this required already? Why not just require ops-scenario when running testing, and not bother with the skipif decorator (which will then be required on many new tests).

Reply:
It's not required if you specifically run the tests in the top-level `test` folder. If you run both those tests and the ones in `testing/tests` (which the top-level `tox -e unit` does) then it is required (and tox takes care of that for you).

However, I agree that if we're going to start regularly adding tests that use Scenario with the expectation that eventually all the Harness use is replaced (outside test_testing) then a skip marker will be too much noise, and we just require `ops[testing]` to run any of the ops unit tests.